test/e2e/run_test.go

```diff
@@ -233,6 +233,40 @@ var _ = Describe("Podman run", func() {
 		return jsonFile
 	}
 
+	It("podman run mask and unmask path test", func() {
+		session := podmanTest.Podman([]string{"run", "-d", "--name=maskCtr1", "--security-opt", "unmask=ALL", "--security-opt", "mask=/proc/acpi", ALPINE, "sleep", "200"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		session = podmanTest.Podman([]string{"exec", "maskCtr1", "ls", "/sys/firmware"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.OutputToString()).To(Not(BeEmpty()))
+		Expect(session.ExitCode()).To(Equal(0))
+		session = podmanTest.Podman([]string{"exec", "maskCtr1", "ls", "/proc/acpi"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.OutputToString()).To(BeEmpty())
+		Expect(session.ExitCode()).To(Equal(0))
+
+		// session = podmanTest.Podman([]string{"run", "-d", "--name=maskCtr2", "--security-opt", "unmask=/proc/acpi:/sys/firmware", ALPINE, "sleep", "200"})
```

Comment:
Why are these commented out?

Reply:
The tests are passing locally for me but failing in ci here, so was trying to see if I can get one of the tests in this block to pass. Will uncomment once I figure out the issue.